test/blocks/comparison/comparison.test.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable no-unused-expressions */
+/* global describe it */
+
+import { readFile } from '@web/test-runner-commands';
+import { expect } from '@esm-bundle/chai';
+import init from '../../../blocks/comparison/comparison.js';
+import { setLibs } from '../../../scripts/utils.js';
+
+setLibs('/libs');
+document.body.innerHTML = await readFile({ path: './mocks/body.html' });
+describe('Comparison Table', () => {
+  it('Verify Comparison Table Exists', async () => {
+    const comparison = document.querySelector('.comparison');
+
+    await init(comparison);
+
+    expect(comparison.querySelector('table')).to.exist;
+    expect(comparison.querySelector('thead tr th')).to.exist;
+    expect(comparison.querySelector('tbody tr th')).to.exist;
+    expect(comparison.querySelector('.cta-row')).to.exist;
```

Comment:
Best practice is to have one expect per test, with some exceptions, so that when the test fails you know what part is broken

Reply:
Split this into two tests, one to test the table structure and the second to check the CTA row